File: 3388.py

Total = raw_input()
inT = int(Total)
T = 0
while inT > 0:

    LatotS = ""
    Latot = list(Total)
    print (Latot)
    Latot.sort(key=int)
    print (Latot)
    LatotS = LatotS.join(Latot)
    print (LatotS)

    inT = inT - int(LatotS)
    T = T + 1

print (T)

File: 3388.cpp
// Example program
#include <iostream>
#include <string>
#include <algorithm>
#include <sstream>
#include <string>

using namespace std;

int main()
{

    string Num;
    string Num2;
    long long Total = 1;
    cin >> Num;
    long long n = 0;

    while(Num != "0"){
        Num2 = Num;
        sort(Num2.begin(),Num2.end());
        Num = to_string(stoll(Num) - stoll(Num2));
        n++;
    }

    cout << n << endl;


}
